packages/components/menu/examples/MenuWithIcons.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Menu, IconButton } from '@contentful/f36-components';
+import { MenuIcon, DoneIcon } from '@contentful/f36-icons';
+
+export default function MenuWithIcons() {
+  return (
+    <Menu>
+      <Menu.Trigger>
+        <IconButton
+          variant="secondary"
+          icon={<MenuIcon />}
+          aria-label="toggle menu"
+        />
+      </Menu.Trigger>
+      <Menu.List>
+        <Menu.Item startIcon={<DoneIcon />}>Item 1</Menu.Item>
+        <Menu.Item endIcon={<MenuIcon />}>Item 2</Menu.Item>
+        <Menu.Item startIcon={<DoneIcon />} endIcon={<MenuIcon />}>
+          Item 3
+        </Menu.Item>
```

Comment:
Good question, it wasn't the case for other places we use startIcon and endIcon, but could be the case here
@fabe WDYT??

Reply:
two questions:
- can we replace `MenuIcon` with some more real use-case examples, like? 
- should we set the icon color to the color of the text by default?

Reply:
> two questions:
> 
> * can we replace `MenuIcon` with some more real use-case examples, like?
👍 
> * should we set the icon color to the color of the text by default?

I didn't do that because it could be used for other icons too, similar to the apps menu:
<img width="262" alt="image" src="https://user-images.githubusercontent.com/1071799/231697517-6fac578a-e679-45fa-ae21-3fe4a85e49e1.png">

Reply:
or maybe even let's remove it completely since @fabe says there are no use cases for the end icon. I think we will introduce `keyboard shortcut hints` as a different prop, not by using `endIcon`, so might be safer to save that space for the future.

Reply:
I'll keep it as icon for now, and only render it on the left side, and we can see what we don with the right side later on